core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -109,7 +116,16 @@ export class CkEditor4RteComponent implements AfterViewInit, OnDestroy {
           name: 'document',
           items: ['Source']
         }
-      ]
+      ],
+      format_tags: 'heading;normal',
```

Comment:
Did `format_tags` already exist in the type for `CKEDITOR.config`?

Reply:
yes, format_tags is already a property(index.ts of cke editor):
![Screenshot from 2021-07-17 23-55-25](https://user-images.githubusercontent.com/68547101/126046382-1b546027-86c6-48ce-9506-a46b911ed74b.png)